### PR TITLE
feat: add README

### PR DESCRIPTION
**Pull request by Zyborg777**
Created a readme for the Manager and added useful info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# ReVanced Manager
+💊 **The official ReVanced Manager**
+## What is ReVanced Manager?
+**ReVanced Manager is a GUI app that will let you build and patch other apps such as YouTube, YouTube Music, Reddit, Twitter, TikTok and more!**
+> **Our goal is to make it as easy and friendly as possible so everyone can use it :D**
+
+## Information
+
+### Status
+🟡 ReVanced Manager is currently in WiP (Work in Progress), which means that it's not ready yet to be used by everyone as we're still coding the app and adding plenty of things.
```
Comment (Ushie) [inline README.md:10]:
```suggestion
🟡 ReVanced Manager is currently WIP (Work in Progress), usage of the app is not recommended as it is not complete and is under active development. 
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# ReVanced Manager
+💊 **The official ReVanced Manager**
+## What is ReVanced Manager?
+**ReVanced Manager is a GUI app that will let you build and patch other apps such as YouTube, YouTube Music, Reddit, Twitter, TikTok and more!**
```
Comment (Sculas) [inline README.md:4]:
```suggestion
**ReVanced Manager is an Android app that will let you patch other apps such as YouTube, YouTube Music, Reddit, Twitter, TikTok, and more!**
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+> **Our goal is to make it as easy and friendly as possible so everyone can use it :D**
```
Comment (Sculas) [inline README.md:5]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+## Information
+
+### Status
+🟡 ReVanced Manager is currently WIP (Work in Progress), usage of the app is not recommended as it is not complete and is under active development. 
+#### ❔ Should I try building it?
+> At the moment, we don't recommend normal users to build it as the code is quite unstable and we're adding new things by the minute, so it's always changing. The only exception, of course, is if you really want to help us and contribute to the project, in which we'd be really grateful!
+
+**Normal users, please hang on tight and wait until the Manager is fully released! I promise you won't be disappointed ;)**
```
Comment (Sculas) [inline README.md:14]:
```suggestion
```

**Review by Sculas — CHANGES_REQUESTED** [review]

**Comment by Sculas** [comment]
Maybe add a note for the Flutter branch?

**Pull request merged**